config/locales/doc_auth/es.yml

```diff
@@ -264,9 +264,9 @@ es:
         modificar su información.
       selfie_capture_status:
         face_close_to_border: DEMASIADO CERCA DEL MARCO
-        face_not_found: NO SE ENCONTRÓ LA CARA
-        face_too_small: LA CARA ES DEMASIADO CHICA
-        too_many_faces: HAY DEMASIADAS CARAS
+        face_not_found: NO SE DETECTÓ EL ROSTRO
```

Comment:
Looks like this is existing, but wonder if your team would consider reevaluating these texts as all-caps, as it can have a number of accessibility challenges.

Nicely compiled in this comment: https://github.com/humanmade/hm-pattern-library/issues/75#issuecomment-235672885

Reply:
@aduth , I belive we kind of discussed, these texts are to customize Acuant SDK hints, and the SDK has known issues with SDK reader, even we have normal texts. @kellular, please correct me if I am wrong.

PS. there are discussions on how to fix the screen reader issue on our side in Timnit.

On the other hand, it still make sense to use normal text to avoid additional work down the road once we fix the SDK issue.

Reply:
@aduth , I agree with your assessment, just wondering whether our designer already had kind of plan for it.